app/src/main/java/com/example/henzoshimada/feeltrip/MainScreen.java

```diff
@@ -197,6 +239,25 @@ private void addItemsOnEmotionalStateSpinner(){
                 android.R.layout.simple_spinner_item, emotionalStateList);
         emotionalStateAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
         emotionalStateSpinner.setAdapter(emotionalStateAdapter);
+        emotionalStateSpinner.setOnItemSelectedListener(new AdapterView.OnItemSelectedListener()
+        {
+            public void onItemSelected(AdapterView<?> parent, View view, int position, long id)
+            {
+                String selectedItem = parent.getItemAtPosition(position).toString();
```

Comment:
I think the spinner will be staying, I cant think of a better implementation  as of yet.